### PR TITLE
initial terraform setup and rds setup

### PR DESCRIPTION
**Pull request by parthshahva**

### CODE REVIEW

File: infra/Makefile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+default: plan
+
+init:
+	cd envs/$(DEPLOYMENT_STAGE) && make init
```
Comment (Bento007) [inline infra/Makefile]: Nit: I like this syntax more.
```suggestion
	make -C envs/$(DEPLOYMENT_STAGE) init
```

File: infra/Makefile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+default: plan
```
Comment (Bento007) [inline infra/Makefile]: why not just reuse this https://github.com/HumanCellAtlas/fusillade/blob/b5ec9ed8fa3a45fc0f449ed01b828c2bbafc80a5/infra/Makefile#L31-L48

Comment (parthshahva) [reply infra/Makefile]: What is missing in my version of the makefile?

File: infra/Makefile (hunk 1 above)
Comment (parthshahva) [reply infra/Makefile]: Leaving as is. Feel free to change on another PR if you have strong feelings

**Pull request merged**